src/docs/app-monitoring/best-practices-for-application-logging-in-openshift.md

```diff
@@ -17,51 +17,54 @@ content_owner: Olena Mitovska
 
 Most of this document will be about logging in Openshift. If you’re looking for general logging best practices while developing your application, here are some helpful resources:
 
-
 * [Heroku Dev Center - Writing best practices for application logs](https://devcenter.heroku.com/articles/writing-best-practices-for-application-logs)
 * [NewRelic - logging best practices](https://newrelic.com/blog/best-practices/best-log-management-practices)
 * [DataSet - the 10 commandments of logging](https://www.dataset.com/blog/the-10-commandments-of-logging/)
 * [Splunk - logging best practices](https://dev.splunk.com/enterprise/docs/developapps/addsupport/logging/loggingbestpractices/)
 
+## Console logs in Loki
 
-## Console logs in Kibana
 In OpenShift, you can easily access application console logs through the web console or the OpenShift CLI. These logs are accessible as long as the container is running, but keep in mind that restarting the container erases logs from its previous instance.
 
 Pods might restart to relocate from one node to another, causing logs to vanish from OpenShift unexpectedly. Stay aware that logs are tied to the container's lifecycle, and a container restart means a fresh log slate.
 
-Kibana serves as a centralized tool that gathers log messages, known as "console logs," from the standard output and standard error streams of your application. It covers all pods running on the OpenShift platform, preserving these logs for approximately 14 days. While the retention period may vary slightly, any adjustment will not exceed 24 hours.
+Loki serves as a centralized tool that gathers log messages, known as "console logs," from the standard output and standard error streams of your application. It covers all pods running on the OpenShift platform, preserving these logs for 14 days.
 
 Should your team require log storage beyond the 14-day window, it's your responsibility to implement a solution. The Platform Team recommends leveraging the [OCIO Object Storage service](../platform-architecture-reference/platform-storage.md) for efficient log storage. For additional details, refer to the next section on logs and long-term log retention.
 
-### Accessing Kibana with OpenShift credentials
-To access Kibana, use your OpenShift credentials. If you can view a pod's console logs in the OpenShift web console, you can seamlessly access the same pod's logs in Kibana. A minimal setup in Kibana is required for log viewing and querying, especially if you're new to Kibana.
+### Accessing Loki
+
+Loki is built into the OpenShift web console. You can access it under the side bar section Observe and then selecting Logs.
 
 For detailed instructions on this setup process, refer to our OCP101 lab section [covering logging and visualizations](https://github.com/bcgov/devops-platform-workshops/blob/master/101-lab/content/12_logging_and_visualizations.md). This will guide you through the necessary steps to make the most of Kibana's features.
```

Comment:
```suggestion
For detailed instructions on this setup process, refer to our OCP101 lab section [covering logging and visualizations](https://github.com/bcgov/devops-platform-workshops/blob/master/101-lab/content/12_logging_and_visualizations.md). This will guide you through the necessary steps to make the most of Loki's features.
```